core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -0,0 +1,167 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service that maintains the title, introduction and list of
+ * questions shown in the simple editor. This service provides functionality
+ * for updating the questions that are displayed in the UI, and passing the
+ * updates through to services like explorationStatesService that are
+ * responsible for generally maintaining the frontend exploration data.
+ */
+
+oppia.factory('SimpleEditorManagerService', [
+  'StatesToQuestionsService', 'SimpleEditorShimService',
+  'QuestionObjectFactory',
+  function(
+      StatesToQuestionsService, SimpleEditorShimService,
+      QuestionObjectFactory) {
+    var data = {
```

Comment:
``data`` is really vague. Why not have the service return title, intro HTML, and the question list as separate values, rather than trying to force these all into one object? Is it so they are bindable? I really want to use the word state here, too. :(